pkg/frontend/back_exec.go

```diff
@@ -982,7 +982,7 @@ func (sh *SqlHelper) GetSubscriptionMeta(dbName string) (*plan.SubscriptionMeta,
 }
 
 // Made for sequence func. nextval, setval.
-func (sh *SqlHelper) ExecSql(sql string) (ret []interface{}, err error) {
+func (sh *SqlHelper) ExecSql(sql string) (ret [][]interface{}, err error) {
```

Comment:
it's confused for me to modify the return type.
I saw its usages at file `function / senfunc.go` will only use its first value.

but I do not know this function very much, So it's just a comment here.

Reply:
Thank you for the comment. I have modified all the other usages of ExecSql to only pick the 1st row.